pages/_en-US/ds-index/ds-bios-firmware-dump.md

```diff
@@ -0,0 +1,72 @@
+---
+lang: en-US
+layout: wiki
+section: ds-index
+category: guides
+title: BIOS/Firmware Dumping
+description: How to extract the Nintendo DS or DSi BIOS/Firmware from your console
+tabs:
+  - tab-dsi-sd-card: DSi with custom firmware
+    tab-flashcard: Flashcard
+---
+
+Some emulators, such as melonDS, require the DS-mode or DSi-mode BIOS and firmware, which is only available by extracting from your console. There are two distinct ways to achieve this, being split into a DSi with custom firmware, or a DS, DS Lite, DSi or 3DS without custom firmware and a flashcard compatible with the console.
```

Comment:
```suggestion
Some emulators, such as melonDS, require the DS-mode or DSi-mode BIOS and firmware, which are only available by extracting from your console. There are two distinct ways to achieve this, being split into a DSi with custom firmware, or a DS, DS Lite, DSi, or 3DS without custom firmware and a flashcard compatible with the console.
```
couple more 😅

Reply:
also thinking on it i’m not sure “only available” is the best thing to say, that implies they’re console specific, perhaps “available by” would be better?

Reply:
I am failing to understand that point. It _is_ console-specific; you can't obtain them without a console, no? Unless we're just fully endorsing piracy now

Reply:
console specific means unique to one console, I'm not endorsing it, but I don't like incorrectly stating things don't exist

Reply:
All points addressed; used a list instead of whatever jargon that was earlier